src/components/userCard.tsx

```diff
@@ -0,0 +1,27 @@
+import projectsCardIcon from "../assets/icons/projectsCardIcon.svg";
+import { UserCardProps } from "@/interfaces/userInterface";
+
+export function UserCard(props: UserCardProps) {
+  const { data } = props;
+  const imageUrl = data.imageUrl ?? "";
+
+  return (
+    <div className="bg-userCardBG w-48 h-52 rounded-2xl flex flex-col justify-center items-center shadow-md relative">
```

Comment:
Taking advantage of the other comment about separating a class or component for the avatar, this card is also something the project will use frequently. I suggest doing the same, so we avoid having duplicate code.

Reply:
i created a card that is just for that, thank you soo much for teaching me